apps/storybook/.storybook/util/sortStories.js

```diff
@@ -0,0 +1,36 @@
+// Inspired by: https://github.com/sumup-oss/circuit-ui/blob/598f9630ff049b0768485670d6e6f978660e4072/.storybook/util/story-helpers.js
+const splitStoryName = name => name.split('/');
+
+const position = (item, array = []) => {
```

Comment:
https://ramda-extension.firebaseapp.com/repl/?v=%5E0.26.1&ve=0.10.1#?const%20getPosition%20%3D%20R_.composeC%28%0A%20%20when%28equals%28-1%29%2C%20always%281000%29%29%2C%0A%20%20indexOf%0A%29%3B%0A%0AgetPosition%28%22a%22%2C%20%5B%22b%22%2C%20%22a%22%5D%29%0AgetPosition%28%22a%22%2C%20%5B%22a%22%5D%29

better rename to "getSortPosition"